src/components/app.tsx

```diff
@@ -135,8 +138,11 @@ export const App = () => {
   }
 
   return (
-    <>
+    <ErrorBoundary
```

Comment:
There are a couple blocks above (lines 47 and 58) above that should probably also be wrapped for this to be consistent. I think it would probably make the most sense for now to just put the error boundary around `<App />` in `index.tsx`. That will catch all of these different render paths in one place.